pkg/expr/sql/parser_allow.go

```diff
@@ -138,6 +138,31 @@ func allowedNode(node sqlparser.SQLNode) (b bool) {
 	case *sqlparser.Where:
 		return
 
+	// // New safe node types
+	// case sqlparser.Values, sqlparser.ValTuple:
+	// 	return
+
+	// case *sqlparser.GroupConcatExpr:
+	// 	return
+
+	// case *sqlparser.Default:
```

Comment:
Default cannot be used at Read time with MySQL. Only at table creation and record insertion